### DISCUSSION TITLE
Stable Diffusion is broken : (

### DISCUSSION THREAD

**Discussion by SalmonPete** [ChefKissInc/NootRX#80] [Q&A] [answered]
It's not a problem just making you aware, Neither DiffusionBee or Stable Diffusion from the command line are not working as they should. Similar issue to #71 also, lower than expected bench, previously 175K. Can I access earlier commits from around June 14?, (I tried the version from #71), Stable Diffusion was working in Sequoia I can't remember which commit it was.

![Screenshot 2024-08-26 at 10 43 02 AM](https://github.com/user-attachments/assets/0df15e6a-fd5b-438b-ac73-6e52e3e530a7)

**Comment by VisualEhrmanntraut** [comment]
You are on macOS Sequoia, this is a beta OS and expected to have bugs. In addition, for Sequoia on AMD, there are two pairs of PAT patches; you will need to switch both to use Shaneee's for better performance.

**Reply by VisualEhrmanntraut** [reply]
Also you are using ForgedInvariant on a desktop AMD CPU. Remove it.

**Reply by CraigHazan** [reply]
Removed, thanks. Unfortunately, Stable Diffusion is still broken, but the Metal Bench went up to 95K!. On my 4700G build with latest NootedRed, Stable Diffusion works well under macOS Sequoia. Like you say, Sequoia is still Beta so we can expect things not to work.

**Comment by Jack-49** [comment]
Same here in Sonoma and RX6700XT, Intel CPU.
Also green overlay in a few games (reported in issues section)

**Reply by SalmonPete** [reply, marked as answer]
Stable Diffusion and DiffusionBee will produce bad results because of PyTorch, which ended support for X86 in January 2024. You can roll back to 2.0.2 for x86 support. @VisualEhrmanntraut this thread can be deleted as it's nothing to do with NootRX.
